deploy/sre-prometheus/100-sre-kubequotaexceeded.PrometheusRule.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-kubequotaexceeded
+    role: alert-rules
+  name: sre-kubequotaexceeded
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-kubequotaexceeded
+    rules:
+    - alert: KubeQuotaExceededSRE
+      expr: |
+        kube_resourcequota{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kube-state-metrics", type="used"}
+        / ignoring(instance, job, type)
+        (kube_resourcequota{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kube-state-metrics", type="hard"} > 0)
```

```suggestion
        kube_resourcequota{namespace=~"(^kube$|^kube-.*|^openshift$|^openshift-.*|^default$)",job="kube-state-metrics", type="used"}
        / ignoring(instance, job, type)
        (kube_resourcequota{namespace=~"(^kube$|^kube-.*|^openshift$|^openshift-.*|^default$)",job="kube-state-metrics", type="hard"} > 0)
```
Just recommend tightening up the regex.  Borrowing this from the latest in the rbac permissions operator config for dedicated-admin: https://github.com/openshift/managed-cluster-config/blob/master/deploy/rbac-permissions-operator-config/50-dedicated-admins.SubjectPermission.yaml